promql/engine.go

```diff
@@ -41,6 +41,26 @@ const (
 	minInt64 model.SampleValue = -9223372036854775808
 )
 
+var (
+	currentQueries = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: namespace,
+		Subsystem: subsystem,
+		Name:      "queries",
+		Help:      "The current number of queries being executed or waiting.",
+	})
+	maxConcurrentQueries = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: namespace,
+		Subsystem: subsystem,
+		Name:      "max_concurrent_queries",
```

Comment:
The naming here should follow that of the other one as a suffix, so queries_concurrent_max would be better.